File: Brainz/Model/Area.cs
namespace Brainz.Model
{
    using Newtonsoft.Json;
    using System;

    public class Area
    {
        [JsonProperty("name")]
        public string Name { get; set; }

        [JsonProperty("sort-name")]
        public string SortName { get; set; }

        [JsonProperty("iso-3166-1-codes")]
        public string[] Iso31661Codes { get; set; }

        [JsonProperty("id")]
        public Guid Id { get; set; }

        [JsonProperty("disambiguation")]
        public string Disambiguation { get; set; }

        [JsonProperty("type")]
        public string Type { get; set; }

        [JsonProperty("type-id")]
        public Guid TypeId { get; set; }

        [JsonProperty("life-span")]
        public AreaLifeSpan LifeSpan { get; set; }
    }
}

File: go/src/model/area.go
package model

type Area struct {
	ID       string   `json:"id"`
	Type     string   `json:"type"`
	TypeID   string   `json:"type-id"`
	Name     string   `json:"name"`
	SortName string   `json:"sort-name"`
	LifeSpan Lifespan `json:"life-span"`
}
